netbox/users/api/serializers.py

```diff
@@ -52,6 +52,18 @@ def create(self, validated_data):
 
         return user
 
+    def update(self, instance, validated_data):
+        """
+        Ensure proper updated password hash generation.
+        """
+        password = validated_data.pop('password', None)
+        if password is not None:
+            instance.set_password(password)
+
+        instance.save()
+
+        return instance
```

Comment:
We're missing a call to `super().update()` here, which is needed for additional validation and to handle any future many-to-many relationships.

Reply:
Removed `instance.save()` in favor of `super().update()` as instance is saved on super method.